release-notes.txt

```diff
@@ -1,3 +1,619 @@
+* Can't build relic with benchmarks or tests (#12897)
+* CC2538DK board docs: broken links (#12889)
+* cpu/stm32/periph/rtc overflow error (#16574)
+* cpu/stm32: some tests are failing on CM33 (l5, u5) (#17439)
+* doc/LOSTANDFOUND: not rendered as expected (#17063)
+* edbg: long lines flooded over serial become garbled (#14548)
+* examples / tests: LoRa tests fail on platforms that don't support
+  LoRa (#14520)
+* feather-m0: `make flash` reports "device unsupported" (#17722)
+* flashing issue on frdm-k64f (#15903)
+* frdm-k22f failing tests/periph_flashpage (#17057)
+* I2C not working under RIOT with U8G2 pkg (#16381)
+* ieee802154_security: Nonce is reused after reboot (#16844)
+* lwip: drivers/at86rf2xx/at86rf2xx_netdev.c invalid state during TCP
+  disconnect (#17209)
+* lwip: invalid state transition on ieee802154_submac users (#17208)
+* Making the newlib thread-safe (#4488)
+* mcuboot: flashes but no output (#17524)
+* nanocoap: incomplete response to /.well-known/core request (#10731)
+* newlib-nano: Printf formatting does not work properly with `"PRIu8"` (#17083)
+* Order of auto_init functions (#13541)
+* pkg/tinydtls: Multiple issues (#16108)
+* pkg_libhydrogen tests fail / update libhydrogen (#18508)
+* Potential race condition in compile_and_test_for_board.py (#12621)
+* RIOT is saw-toothing in energy consumption (even when idling) (#5009)
+* riotboot/nrf52840dk: flashing slot1 with JLINK fails (#14576)
+* riotboot: ECC faults (eg. in STM32L5 or STM32WB) not handled
+  gracefully (#17874)
+* rust-gcoap example is incompatible with littlefs2 (#17817)
+* SPI access in LoRa-e5 and im880b? (#19025)
+* stdio_ethos: infinite shell loop (#17972)
+* stdio_tinyusb_cdc_acm hangs with picolibc (#19277)
+* sys/riotboot: documentation issues (#11243)
+* sys/stdio_uart: dropped data when received at once (#10639)
+* tests/cpp11_*: failing on i-nucleo-lrwan1 (#14578)
+* tests/lwip target board for python test is hardcoded to native (#6533)
+* tests/periph_flashpage: failing on stm32l475ve (#17280)
+* tests/pkg_libhydrogen: test fails on master for the samr21-xpro with
+  LLVM (#15066)
+* tests/pkg_libschc: Failing test_reassemble_success_ack_always (#19445)
+* tests/test_tools: test fails while testing on samr21-xpro/iotlab-m3 (#15888)
+* tests: broken with stdio_rtt if auto_init is disabled (#13120)
+* tests: some tests don't work with `newlib` lock functions. (#12732)
+* usbus/msc: wrong error handling and behavior after usb reset (#19478)
+* Use of multiple CAN bus on compatible boards (#14801)
+
+There are 156 known issues in this release
+
+Fixed Issues since the last release (2023.01)
+=============================================
+
+- bootloader: BTN_BOOTLOADER_INVERTED doesn't work as expected. (#19364)
+- Race condition in SUIT around threads terminating (#19195)
+- socket_zep: fragmentation broken if destination is multicast /
+  ACK_REQ not set (#19117)
+
+3 fixed issues since last release (2023.01)
+
```

Comment:
```suggestion

Boards tested during this release
=================================

- `esp32-wroom-32`    ✅ PASS
- `nrf52840dk`        ✅ PASS
- `nucleo-f429zi`     ✅ PASS
- `samr21-xpro`       ✅ PASS
- `arduino-mega2560`  ❌ FAIL
- `b-l072z-lrwan1`    ❌ FAIL
- `b-l475e-iot01a`    ❌ FAIL
- `dwm1001`           ❌ FAIL
- `frdm-kw41z`        ❌ FAIL
- `iotlab-m3`         ❌ FAIL
- `nrf52dk`           ❌ FAIL
- `nucleo-l452re`     ❌ FAIL
- `nucleo-wl55jc`     ❌ FAIL
- `remote-revb`       ❌ FAIL
- `saml10-xpro`       ❌ FAIL
- `samr34-xpro`       ❌ FAIL

See the release tracking issue (#19469) for more details.
```

Reply:
How did those fail?

Reply:
We never actually test the boards, I expect we will spend the next release cycle tracking down all the issues, many probably to do with the test setup more than anything.

Reply:
The cool thing is that some boards pass, mostly the ones that are tested regularly due to the Kaspars PI Fleet.

Reply:
The list of boards failing seems to come from the test-on-iotlab github actions workflow.

Reply:
Actually, the RC3 was tested on iotlab: https://github.com/RIOT-OS/RIOT/actions/runs/4787593567

I see that `tests/pkg_libschc` is failing on lora boards, for some reason. Maybe it requires some setup ? If that's the case, then it's a false positive because there are missing bits in the build system (tests script should in a `test-with-config` directory). I haven't checked the others in details but at least there are logs :)

Reply:
> I see that `tests/pkg_libschc` is failing on lora boards, for some reason. Maybe it requires some setup ? If that's the case, then it's a false positive because there are missing bits in the build system (tests script should in a `test-with-config` directory)

Or maybe it's just that the unittests are broken:
```
ERROR: test_reassemble_success_ack_always (__main__.TestSelfTest)
```

Reply:
@aabadie see https://github.com/RIOT-OS/RIOT/issues/19445

Reply:
Also, the test does not use any connectivity but just checks the output of libschc, so I don't think it is a bootstrapping issue... or that the board needs some additional config. The fact is, however, that libSCHC is still a bit flaky in general.

Reply:
> @aabadie see #19445

Thanks, I missed the fact that it's a known issue :)